server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -167,11 +167,14 @@ public enum MetadataVersion {
     // and updates to a handful of RPCs.
     IBP_3_4_IV0(8, "3.4", "IV0", true),
 
-    // Support for tiered storage (KIP-405) and SCRAM
+    // Support for tiered storage (KIP-405)
     IBP_3_5_IV0(9, "3.5", "IV0", false),
 
     // Adds replica epoch to Fetch request (KIP-903).
-    IBP_3_5_IV1(10, "3.5", "IV1", false);
+    IBP_3_5_IV1(10, "3.5", "IV1", false),
+
+    // Support for SCRAM 
+    IBP_3_5_IV2(11, "3.5", "IV2", false);
```

Comment:
didMetadataChange needs to be `true` here

Reply:
Fixed! Hopefully it will build and auto merge.